day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,29 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings = toppings
+  end
+
+  def remove_topping
+    @toppings = nil
```

Comment:
How would you rewrite this so that you remove whatever topping gets passed as an argument?